test/pool.test.ts

```diff
@@ -1119,6 +1219,77 @@ describe("EthPool", function () {
     // Receiver had 5 blocks paying 1 per block
     await receiver.collect(5);
   });
+
+  it("Allows full sender update with top up", async function () {
+    const [sender, proxy, receiver1, receiver2] = await getEthPoolUsers();
+    const proxyWeightBase = await sender.pool.PROXY_WEIGHTS_SUM();
+    await proxy.setProxyWeights([[receiver2, proxyWeightBase]]);
+    await sender.updateSender(
+      0,
+      proxyWeightBase * 6,
+      proxyWeightBase * 3,
+      [[receiver1, proxyWeightBase]],
+      [[proxy, proxyWeightBase * 2]]
+    );
+    await mineBlocks(1);
+    await mineBlocksUntilCycleEnd();
```

Comment:
Why do we have two mine statements? Can we add a comment for this or somehow abstract it away?

Reply:
Good catch, removed. It makes no sense since `mineBlocksUntilCycleEnd` always mines at least 1 block.